incubator/command-dump/src/test/java/io/aklivity/zilla/runtime/command/dump/internal/airline/ZillaDumpCommandTest.java

```diff
@@ -195,34 +297,40 @@ public void init()
         command.verbose = true;
         command.continuous = false;
         command.properties = List.of(String.format("zilla.engine.directory=%s", Paths.get(baseDir, "engine")));
-        command.output = Paths.get(tempDir.getPath(), "test.pcap");
+        command.output = Paths.get(tempDir.getPath(), "actual.pcap");
     }
 
     @Test
-    public void shouldDumpWithoutFilter() throws IOException
+    public void shouldWritePcap() throws IOException
     {
+        // GIVEN
+        File expectedDump = new File(baseDir + "/expected_dump.pcap");
+        byte[] expected = Files.readAllBytes(expectedDump.toPath());
```

Comment:
This should be loaded from the classpath (logical) instead of filesystem (physical) because `src/test/resources` are included in `target/test-classes`.

Same feedback for all files loaded via `baseDir`.

Reply:
👍